pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
+	noMsg := ""
+	client, err := t.getDestinationClient()
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	analytic := migapi.MigAnalytic{}
+
+	err = client.Get(context.Background(),
+		k8sclient.ObjectKey{
+			Name:      t.PlanResources.MigPlan.Name,
+			Namespace: t.PlanResources.MigPlan.Namespace,
+		},
+		&analytic)
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	missingArtifactsInNamespace := make(map[string]string, 0)
+	for _, ns := range analytic.Status.Analytics.Namespaces {
+		missingArtifacts := make([]string, 0)
+		if ns.PVCount == 0 {
+			missingArtifacts = append(missingArtifacts, "Persistent Volumes")
+		}
+		if ns.ImageCount == 0 {
+			missingArtifacts = append(missingArtifacts, "Images")
+		}
+		if ns.K8SResourceTotal == 0 || ns.K8SResourceTotal == ns.ExcludedK8SResourceTotal {
```

Comment:
Also, i don't think you want to compare `K8SResourceTotal` with `ExcludedK8SResourceTotal` -- if a resource is in excluded total, then it won't be added to total. I think the comparison to 0 is all you need here.

Reply:
@sseago This comment and related ones (w.r.t MigAnalytics) might not be applicable following Derek's comments. So, shall I make any changes to this PR based on what we decide based on further discussion?